packages/babel-core/test/api.js

```diff
@@ -648,14 +648,14 @@ describe("api", function () {
     });
   });
 
-  it("default source map filename", function () {
-    return transformAsync("var a = 10;", {
+  it("default source map filename", async function () {
+    const result = await transformAsync("var a = 10;", {
       cwd: "/some/absolute",
       filename: "/some/absolute/file/path.js",
       sourceMaps: true,
-    }).then(function (result) {
-      expect(result.map.sources).toEqual(["path.js"]);
     });
+
+    expect(result.map.sources).toEqual(["file/path.js"]);
```

Comment:
I'm not sure if we should consider this PR as breaking because of this change, or as a bugfix.